development/src/composables/dbService.js

```diff
@@ -0,0 +1,82 @@
+import { appFirestore, appAuth, app } from "./firebaseConfig/config";
```

Comment:
Isn't app and appAuth the same thing?

Reply:
Yea great spot out
I’ve updated that now.